tasks/managed/update-trusted-tasks/update-trusted-tasks.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-trusted-tasks
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update trusted-tasks list bundles
+  params:
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot json reside
+  steps:
+    - name: update-trusted-tasks
+      image: quay.io/konflux-ci/appstudio-utils@sha256:0901e46d9a41d0c564b0efab092f37cd187bd8b4ff2455ec84539ceb8b08f3ea
+      script: |
+        #!/usr/bin/env bash
+        set -eux
+        TAG="latest"
+
+        SNAPSHOT_SPEC_FILE="$(workspaces.data.path)/$(params.snapshotPath)"
+        if [ ! -f "${SNAPSHOT_SPEC_FILE}" ] ; then
+            echo "No valid snapshot file was found."
+            exit 1
+        fi
+
+        application=$(jq -r '.application' "${SNAPSHOT_SPEC_FILE}")
+        NUM_COMPONENTS=$(jq '.components | length' "${SNAPSHOT_SPEC_FILE}")
+        printf 'Beginning "%s" for "%s"\n\n' "$(context.task.name)" "$application"
+        for ((i = 0; i < NUM_COMPONENTS; i++))
+        do
+            component=$(jq -c --argjson i "$i" '.components[$i]' "${SNAPSHOT_SPEC_FILE}")
+            repository=$(jq -r '.repository' <<< "$component")
+            imageTag=$(jq '.tags' <<< "$component" | jq -r '.[0]')
+            ACCEPTABLE_BUNDLES=$(echo "${repository}" | awk -F'/' '{$NF="data-acceptable-bundles"; print $0}' OFS='/')
+            set +e
+            skopeo list-tags docker://"${ACCEPTABLE_BUNDLES}" | jq -r '.Tags[]' | grep ${TAG} &> /dev/null
```

Comment:
How about doing `skopeo inspect docker://"${ACCEPTABLE_BUNDLES}:latest"` directly?

Reply:
I tried using skopeo inspect, but it is unsupported for this OCI artifact.
```
$ skopeo inspect docker://quay.io/avi_test/user-ns2/release-catalog/data-acceptable-bundles:latest
FATA[0001] unsupported image-specific operation on artifact with type "application/vnd.unknown.config.v1+json"
```

I then switched to skopeo list-tags and checking if `latest` exists

Reply:
I see, I didn't realize this was an oci artifact. Then `oras resolve`? We use it for the same purpose here: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/managed/push-snapshot/push-snapshot.yaml#L61

My worry is that listing all tags can be much heavier. But maybe in this case you only use latest, so probably not really a concern? So I leave it up to you.

Reply:
I understand your concern, but as you mentioned we only use latest.
I think we will keep it as it is now, and in time, if we will see it causes issues (unlikely) we can switch to oras

Reply:
ok, sounds good.